### DISCUSSION TITLE
b-table: highlight row on focus with arrow keys

### DISCUSSION THREAD

**Discussion by mungra** [bootstrap-vue/bootstrap-vue#6182] [Q&A]
Is there a way to highlight rows when using the arrow keys. something similar to the hover event? 

_Originally posted by @mungra in https://github.com/bootstrap-vue/bootstrap-vue/issues/2869#issuecomment-742732155_

**Comment by emfluenceindia** [comment]
Most probably there is no inbuilt solution available for this yet. In my opinion, we need to write our own Keydown handlers.
However, by doing this, we would lose the mouse click control. so, to keep that behaviour alive as well, we need to have a Click event listener call the Keydown triggers from there.

**Comment by Hiws** [comment]
> Is there a way to highlight rows when using the arrow keys. something similar to the hover event?
> 
> _Originally posted by @mungra in [#2869 (comment)](https://github.com/bootstrap-vue/bootstrap-vue/issues/2869#issuecomment-742732155)_

What is it you're trying to accomplish?

**Reply by mungra** [reply]
trying to get the same functionality as with buefy table row selection with arrow keys : [https://buefy.org/documentation/table/](https://buefy.org/documentation/table/)

**Reply by Hiws** [reply]
`<b-table>` in selection mode does have support for using arrow keys, but it wont automatically switch selection when doing so, which I'm guessing is what you want.

To my knowledge, there's currently no built-in way to accomplish this. 
So you would most likely have to implement this yourself. Maybe someone here will have an idea as to how.

**Reply by jacobmllr95** [reply]
See the [Row select support](https://bootstrap-vue.org/docs/components/table#row-select-support) example in the `<b-table>` documentation.
When in `single` mode, you can use the arrow keys to "highlight" a row an use the ENTER key to select it.
